### PR TITLE
Fixed grammar errors in the docs

### PR DESCRIPTION
**Pull request by rocketbear27**
### Requirements for Contributing Documentation

* Fill out the template below. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* The pull request must only contribute documentation (for example, markdown files or API docs). To contribute other changes, you must use a different template. You can see all templates at https://github.com/atom/atom/tree/master/.github/PULL_REQUEST_TEMPLATE.

### Description of the Change

_I fixed some grammar errors that were located in the documentation files. This will make the documentation more clear to users._

### Release Notes

Please describe the changes in a single line that explains this improvement in
terms that a user can understand.  This text will be used in Atom's release notes.

_Fixed grammar errors in the documentation._

If this change is not user-facing or notable enough to be included in release notes
you may use the strings "Not applicable" or "N/A" here.

_N/A_

-----
[View rendered CODE_OF_CONDUCT.md](https://github.com/rocketbear27/atom/blob/master/CODE_OF_CONDUCT.md)
[View rendered CONTRIBUTING.md](https://github.com/rocketbear27/atom/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ This Code of Conduct applies both within project spaces and in public spaces whe
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at [atom@github.com](mailto:atom@github.com). All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+Instances of abusive, harassing or otherwise, unacceptable behavior may be reported by contacting the project team at [atom@github.com](mailto:atom@github.com). All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (lee-dohm) [inline CODE_OF_CONDUCT.md]: This change is incorrect. There are three types of behavior listed here:

* abusive
* harassing
* otherwise unacceptable

This is not intended to be "abusive unacceptable behavior, which may be harassing or otherwise".

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ Here's a list of the big ones:
 * [status-bar](https://github.com/atom/status-bar) - the status bar at the bottom of the UI.
 * [markdown-preview](https://github.com/atom/markdown-preview) - the rendered markdown pane item.
 * [settings-view](https://github.com/atom/settings-view) - the settings UI pane item.
-* [autocomplete-plus](https://github.com/atom/autocomplete-plus) - autocompletions shown while typing. Some languages have additional packages for autocompletion functionality, such as [autocomplete-html](https://github.com/atom/autocomplete-html).
+* [autocomplete-plus](https://github.com/atom/autocomplete-plus) - auto-completions shown while typing. Some languages have additional packages for autocompletion functionality, such as [autocomplete-html](https://github.com/atom/autocomplete-html).
```
Comment (lee-dohm) [inline CONTRIBUTING.md]: This is an unnecessary change, especially when the name of the package is `autocomplete` without a hyphen.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -219,7 +219,7 @@ Please follow these steps to have your contribution considered by the maintainer
 2. Follow the [styleguides](#styleguides)
 3. After you submit your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) are passing <details><summary>What if the status checks are failing?</summary>If a status check is failing, and you believe that the failure is unrelated to your change, please leave a comment on the pull request explaining why you believe the failure is unrelated. A maintainer will re-run the status check for you. If we conclude that the failure was a false positive, then we will open an issue to track that problem with our status check suite.</details>
 
-While the prerequisites above must be satisfied prior to having your pull request reviewed, the reviewer(s) may ask you to complete additional design work, tests, or other changes before your pull request can be ultimately accepted.
+While the prerequisites above must be satisfied prior to have your pull request reviewed, the reviewer(s) may ask you to complete additional design work, tests, or other changes before your pull request can be ultimately accepted.
```
Comment (lee-dohm) [inline CONTRIBUTING.md]: The original is correct.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -309,7 +309,7 @@ All JavaScript must adhere to [JavaScript Standard Style](https://standardjs.com
 
 - Include thoughtfully-worded, well-structured [Jasmine](https://jasmine.github.io/) specs in the `./spec` folder.
 - Treat `describe` as a noun or situation.
-- Treat `it` as a statement about state or how an operation changes state.
+- Treat `it` as a statement about the state or how an operation changes state.
```
Comment (lee-dohm) [inline CONTRIBUTING.md]: The original is common parlance.

Comment (calebmeyer) [reply CONTRIBUTING.md]: Also this is inconsistent. If you want to talk about "the state", then it should be "the state" everywhere, not "the state" for the first instance and "state" for the next.
